lab_1/scripts/update-pipeline-definition.sh

```diff
@@ -0,0 +1,86 @@
+if ! command -v jq  &> /dev/null
+then
+    echo "jq could not be found, but can be easely installed with sudo"
+    exit 0
+fi
+
+if [[ !($1 =~ pipeline\.json$) ]]
+then
+echo "Error: no path to pipeline file is provided"
+exit 1
+fi
+
+if [[ $(jq '.pipeline.version' $1) == null ]]
+then
+    echo "Error: no .version field"
+    exit 1
+fi
+if [[ $(jq '.pipeline.stages[1].actions[0].configuration.EnvironmentVariables' $1) == null ]]
+then
+    echo "Error: no .EnvironmentVariables field for stage 1"
+    exit 1
+fi
+if [[ $(jq '.pipeline.stages[3].actions[0].configuration.EnvironmentVariables' $1) == null ]]
```

Comment:
this solution doesn't work for the req part "in each action"

"The EnvironmentVariables properties in each action are filled with a stringified JSON object containing the BUILD_CONFIGURATION value from the --configuration parameter flag."